.gitignore

```diff
@@ -4,8 +4,6 @@ node_modules/
 .vdiff/
 components/*/test/screenshots/current/
 components/*/test/screenshots/golden/
-helpers/test/screenshots/current/
-helpers/test/screenshots/golden/
```

Comment:
This is the only `vdiff` test in `helpers`, so we can remove this. You'll need to clean up any lingering local copies next time you pull after this merges.